index.html

```diff
@@ -8,61 +8,98 @@
     <title>Company Site</title>
     <META NAME="ROBOTS" CONTENT="NOINDEX, NOFOLLOW" />
   </head>
+
   <body>
-    <nav>
-      <a class="logo" href="#">
+    <header class="header">
+      <a class="header--logo" href="#">
```

Comment:
Used the "--" double-dash notation, makes it a bit easier to see what parent component an element belongs to